server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -206,6 +207,11 @@ public void define(WebService.NewController controller) {
         "Accepted units are 'y' for year, 'm' for month, 'w' for week and 'd' for day. " +
         "If this parameter is set, createdAfter must not be set")
       .setExampleValue("1m2w (1 month 2 weeks)");
+    action.createParam(SINCE_LEAK_PERIOD)
+      .setDescription("To retrieve issues created since the leak period.<br>" +
+        "If this parameter is set to a truthy value, createdAfter must not be set and one component id or key must be provided.")
```

Comment:
Hum, ok I think you should remove this "If this parameter is set to a truthy value", it's misleading as we know it's a boolean value.